src/npm/__tests__/index.test.js

```diff
@@ -47,30 +47,42 @@ describe('getDependents()', () => {
     expect(angular).toBe(0);
     expect(holmes).toBe(0);
   });
+});
 
-  it.skip('has the right approximate value', () => {
-    const [jest, angular, holmes] = dependents.map(pkg =>
-      pkg.dependents.toString()
-    );
-
-    // eslint-disable-next-line no-console
-    console.log('dependents', { jest, angular, holmes });
-
-    // real should be 2100
-    expect(jest).toHaveLength(4);
+describe('getDownload()', () => {
+  it('should download one package and return correct response', async () => {
+    const dl = await api.getDownload('jest');
+    expect(dl.body).toHaveProperty('jest');
+    expect(dl.body.jest).toEqual({
+      downloads: expect.any(Number),
+      start: expect.any(String),
+      end: expect.any(String),
+      package: 'jest',
+    });
+  });
 
-    // real should be 5200
-    expect(angular).toHaveLength(4);
+  it('should download one scoped package and return correct response', async () => {
+    const dl = await api.getDownload('@angular/core');
+    expect(dl.body).toHaveProperty('@angular/core');
+    expect(dl.body['@angular/core']).toEqual({
+      downloads: expect.any(Number),
+      start: expect.any(String),
+      end: expect.any(String),
+      package: '@angular/core',
+    });
+  });
 
-    // real should be 0
-    expect(holmes).toHaveLength(1);
+  it('should download 2 packages and return correct response', async () => {
+    const dl = await api.getDownload('jest,holmes.js');
```

Comment:
should we not have an api which accepts an array, and keep the handling with adding the extra comma?

Reply:
Oh I'm seeing now this is a new private import, seems fine